ethereum/WebaverseVoucher.sol

```diff
@@ -75,10 +73,9 @@ contract WebaverseVoucher is EIP712Upgradeable {
                 keccak256(
                     abi.encode(
                         keccak256(
-                            "NFTVoucher(uint256 tokenId,string metadataurl,uint256 balance,uint256 nonce,uint256 expiry)"
+                            "NFTVoucher(uint256 tokenId,uint256 balance,uint256 nonce,uint256 expiry)"
```

Comment:
We should keep the URL, it just shouldn't be called "metadataurl". It's just the token's content URL, which will most likely be a metaversefile directory. But it could also be a GH pages or similar.

Reply:
Reverted token url back. It's named as `contenturl`.